### PR TITLE
#221: The place order button text changed for PayPal 

### PR DESCRIPTION
**Pull request by artdaw**
#221: The place order button text changed for PayPal to "Confirm and pay with PayPal"

### CODE REVIEW

File: AtlasSDK/AtlasSDK/Models/PaymentMethod.swift [hunk 1]
```diff
@@ -22,4 +22,8 @@ extension PaymentMethod: JSONInitializable {
             metadata: json[Keys.metadata].dictionaryObject)
     }
 
+    public func isPaypal() -> Bool {
+        return self.method?.caseInsensitiveCompare("paypal") == NSComparisonResult.OrderedSame
```
Comment (bonkey) [inline AtlasSDK/AtlasSDK/Models/PaymentMethod.swift]: Pretty sure you can drop `NSComparisonResult` from here. Just `== . OrderedSame`

File: AtlasUI/AtlasUI/CheckoutViewModel.swift [hunk 2]
```diff
@@ -47,6 +47,20 @@ extension CheckoutViewModel {
         return selectedBillingAddress?.fullContactPostalAddress ?? localizer.loc("No Billing Address")
     }
 
+    var submitButtonTitle: String {
+        switch self.checkoutViewState {
+            case .NotLoggedIn: return "Zalando.Checkout"
+            case .CheckoutIncomplete, .LoggedIn:
+                if let paymentMethod = checkout?.payment.selected {
```
Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutViewModel.swift]: `if let paymentMethod = checkout?.payment.selected where paymentMethod.isPaypal() {`

File: AtlasSDK/AtlasSDK/Models/PaymentMethod.swift (hunk 1 above)
Comment (artdaw) [inline AtlasSDK/AtlasSDK/Models/PaymentMethod.swift]: Fixed

File: AtlasUI/AtlasUI/CheckoutViewModel.swift (hunk 2 above)
Comment (artdaw) [inline AtlasUI/AtlasUI/CheckoutViewModel.swift]: Fixed

**Comment by bonkey** [comment]
👍

**Comment by artdaw** [comment]
👍

**Pull request merged**